XCUITests/FxScreenGraph.swift

```diff
@@ -172,6 +180,7 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
         scene.tap(table.cells["Logins"], to: LoginsSettings)
         scene.tap(table.cells["ClearPrivateData"], to: ClearPrivateDataSettings)
         scene.tap(table.cells["OpenWith.Setting"], to: OpenWithSettings)
+        scene.tap(table.cells["Show Tour"], to: Intro_Organize)
```

Comment:
Would it be enough with adding [here](https://github.com/mozilla-mobile/firefox-ios/blob/0fe9935790ce5bf606b3a0e35084608734f6cc3e/Client/Frontend/Settings/AppSettingsOptions.swift#L486) this line:

> override var accessibilityIdentifier: String? { return "ShowTour" }

Or is it also necessary to change something [here](https://github.com/mozilla-mobile/firefox-ios/blob/0fe9935790ce5bf606b3a0e35084608734f6cc3e/Client/Frontend/Settings/AppSettingsOptions.swift#L490 ) to have the same string when calling NSLocalizedString? 
Thanks @jhugman!